reward-center/program/src/state/mod.rs

```diff
@@ -35,19 +45,40 @@ impl RewardCenter {
         1 // bump
     }
 
+    pub fn calculate_total_token_payout(
```

Comment:
```suggestion
fn calculate_total_token_payout(
```

this can be private since used as a helper.